RecoTracker/LSTCore/standalone/code/core/AccessHelper.h

```diff
@@ -9,77 +9,86 @@ enum { kpT5 = 7, kpT3 = 5, kT5 = 4, kpLS = 8 };
 
 // ----* Hit *----
 std::tuple<std::vector<unsigned int>, std::vector<unsigned int>> convertHitsToHitIdxsAndHitTypes(
-    SDL::Event<SDL::Acc>* event, std::vector<unsigned int> hits);
+    SDL::Event<ALPAKA_ACCELERATOR_NAMESPACE::Device>* event, std::vector<unsigned int> hits);
```

Comment:
perhaps a typedef/alias would make `SDL::Event<ALPAKA_ACCELERATOR_NAMESPACE::Device>` more readable/less verbose

Reply:
I was being cautions adding any `using` statements in headers, but having something like this in a header should be fine, right?

```cpp
namespace SDL {
  using namespace ALPAKA_ACCELERATOR_NAMESPACE;

  // code
}
```

Reply:
I actually meant just something like `using SDLEvent = SDL::Event<ALPAKA_ACCELERATOR_NAMESPACE::Device>;` not a `using namespace`

the half-screen width name is just a distracting.